modules/mono/glue/GodotSharp/GodotSharp/Core/Projection.cs

```diff
@@ -0,0 +1,817 @@
+#if REAL_T_IS_DOUBLE
+using real_t = System.Double;
+#else
+using real_t = System.Single;
+#endif
+using System;
+using System.Runtime.InteropServices;
+
+
+namespace Godot
+{
+    /// <summary>
+    /// The parameterless constructor returns <see cref="Projection.Zero"/>, not <see cref="Projection.Identity"/> like GDScript.
+    /// </summary>
```

Comment:
I don't think this comment is necessary, we don't have it for `Transform{2D,3D}`, instead we document it in the `Identity` property and I think that's enough.